functional-tests/functional_tests.go

```diff
@@ -265,9 +267,9 @@ func (this Hoverfly) ProxyWithAuth(r *sling.Sling, user, password string) (*http
 	proxy, _ := url.Parse(fmt.Sprintf("http://%s:%s@localhost:%v", user, password, this.proxyPort))
 	proxyHttpClient := &http.Client{
 		Transport: &http.Transport{
-			Proxy: http.ProxyURL(proxy),
+			Proxy:           http.ProxyURL(proxy),
 			TLSClientConfig: &tls.Config{InsecureSkipVerify: true}},
-			CheckRedirect: func(req *http.Request, via []*http.Request) error { return http.ErrUseLastResponse },
+		CheckRedirect: func(req *http.Request, via []*http.Request) error { return http.ErrUseLastResponse },
```

Comment:
Please check indentations.

Reply:
It is the result of `go fmt`. These lines have different indentations because in the nested struct/object they are in different depth.

Reply:
```
TLSClientConfig: &tls.Config{InsecureSkipVerify: true}},
```
Sorry. I confirmed a nested bracket.